### ISSUE TITLE
This integration causes my Hubitat to have CPU overload and Shutdown the Zigbee radio

### ISSUE THREAD

**Issue by AlohaFlyer** [jason0x43/hacs-hubitat#159]
This integration is working for me, but within a short period of time, it causes severe loads on the HE CPU and shuts down the Zigbee radio. 

Any thoughts on how I can make this more stable? My message queue is filling up and it can't handle it. I did turn off all logging.

**Comment by jason0x43** [comment]
What was the log showing? I have never heard of anything like that happening (although that certainly doesn't mean it isn't), and I'm not sure off the top of my head what might be the problem.

**Comment by AlohaFlyer** [comment]
Thanks for your reply. I will get some logs and upload shortly.

**Comment by jason0x43** [comment]
Try turning on debug logging for the integration in HA. The only thing I can think of offhand is that HA is, for some reason, sending tons of device commands to HE. The integration itself _shouldn't_ be doing that; it receives events from HE and passes them on to HA, and it sends commands to HE when asked to by other parts of HA, but it doesn't send device commands by itself.

> Should I be running an Events Server, configured on the HA side??

The integration starts one up automatically

>  Should I add a URL to POST device events to, on the HE side??

The integration should set that automatically. If you look in the Maker API instance in HE, the POST URL should be set to an address that includes the IP address of your HA instance.

> Do I need to configure this "Enable these hosts (comma separated) for CORS support", on the HE side??

You do not.

> I am basically using my HE as a Zwave/Zigbee Coordinator to feed HA.

That's how I use HE, too 🙂

**Comment by AlohaFlyer** [comment]
Thanks Jason. I did look at HA and I found an automation I had forgotten about that was meant to save energy by turning off of my exterior lights off. I had accidently left it in a mode that was updating about 80 lights every second! lol !  Anyways, I have that disabled and will watch it for a day or so and report back. It was obviously flooding the HE with requests. Thanks for your patience. And thanks for a nice integration.  I need to login and buy you a coffee.
